independent-projects/tools/registry-client/src/main/java/io/quarkus/registry/catalog/json/JsonPlatform.java

```diff
@@ -35,15 +41,31 @@ public void setName(String name) {
     @Override
     @JsonDeserialize(contentAs = JsonPlatformStream.class)
     public List<PlatformStream> getStreams() {
-        return streams == null ? Collections.emptyList() : streams;
+        return streams == null ? Collections.emptyList() : streams.values().stream().collect(Collectors.toList());
```

Comment:
Lets change the interface to return a Collection to avoid creating a list on each call.

Reply:
Ok, this might break current users ? I'll make the change.

Reply:
I am not sure that this is better, and will definitely break current users, although they can just wrap in List if needed. Let me know what you think.